tests/dummy/app/templates/list-controls.hbs

```diff
@@ -28,7 +28,7 @@
         {{#paper-subheader class="md-no-sticky"}}Clickable Items with Secondary Controls{{/paper-subheader}}
 
         {{#paper-item action="transitionToWifiMenu"}}
-          {{paper-icon "network-wifi"}}
```

Comment:
I see. The icon names have underscores.

I wonder why we had dashes before. Did material icons change?